go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/go-logr/zapr v1.2.3
 	github.com/google/go-cmp v0.5.8
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
-	github.com/openshift/api v0.0.0-20220907152121-48d78630feb3
+	github.com/openshift/api v0.0.0-20220927015011-24ee13c10f3b
```

Comment:
Miheer original vendor the merge commit github.com/openshift/api v0.0.0-20220929023536-18c298295790 but look like you took the individual commit. Does that matter?

Reply:
This doesn't matter anymore because rebased pulled in much newer API